src/NuGet.Core/NuGet.Configuration/NuGet.Configuration.csproj

```diff
@@ -20,6 +20,7 @@
   
   <ItemGroup>
     <ProjectReference Include="..\NuGet.Common\NuGet.Common.csproj" />
+    <ProjectReference Include="..\NuGet.Packaging\NuGet.Packaging.csproj" />
```

Comment:
why this new dependency? this might regress some RPS since it might increase module count

Reply:
`SettingsUtility` needs `RevocationMode` in order to centralize the reading and parsing of it there.

Reply:
then might make it part of some common library... or else trigger RPS if there is no regression, then I'm fine.